packages/andive/src/components/card.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import styled, {css} from 'styled-components'
+import PropTypes from 'prop-types'
+
+import * as palette from '../constants/palette'
+
+const rootCss = css`
+  background: white;
+  padding: 16px;
+  border-radius: 4px;
+  border: 1px solid ${palette.hexToRGBA(palette.lightPrimary)};
```

Comment:
```suggestion
  border: 1px solid ${palette.lightPrimary};
```

palette.hexToRGBA is only used to add transparency to the color.